jetty-io/src/main/java/org/eclipse/jetty/io/ByteBufferAccumulator.java

```diff
@@ -187,10 +187,15 @@ public void writeTo(OutputStream out) throws IOException
         }
     }
 
-    @Override
-    public void close()
+    public void reset()
```

Comment:
Why adding this method?
It's just for clarity because it's better to `reset()` and the reuse it, rather than `close()` and reuse it?

Reply:
Yes, just because it looks wrong to do `close()` and then reuse it.